routers/index.go

```diff
@@ -93,4 +93,6 @@ func RegisterRoutes(route *gin.Engine) {
 	route.PUT("/group/:id", middleware.JWTAuthMiddleware(), controllers.EditGroup)
 
 	route.GET("/environment/:name", middleware.JWTAuthMiddleware(), controllers.GetEnvironmentByName)
+
+	route.POST("/environment/:id/edit-field-name/:fieldId", middleware.JWTAuthMiddleware(), controllers.EditEnvironmentFieldName)
```

Comment:
can we make this post request with id and fieldId in req body so that we can add signature as well?